modules/distr-tracing-tempo-install-cli.adoc

```diff
@@ -0,0 +1,196 @@
+// Module included in the following assemblies:
+//
+//* distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-tempo-install-cli_{context}"]
+= Installing the {TempoShortName} by using the CLI
+
+You can install the {TempoShortName} on the command line.
+
+.Prerequisites
+
+* An active OpenShift CLI (`oc`) session by a cluster administrator with the `cluster-admin` role.
++
+[TIP]
+====
+* Ensure that your OpenShift CLI (`oc`) version is up to date and matches your {product-title} version.
+
+* Run `oc login`:
++
+[source,terminal]
+----
+$ oc login --username=<your_username> 
+----
+====
+
+* You are using a supported provider of object storage: link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[Red Hat OpenShift Data Foundation], link:https://min.io/[MinIO], link:https://aws.amazon.com/s3/[Amazon S3], link:https://azure.microsoft.com/en-us/products/storage/blobs/[Azure Blob Storage], link:https://cloud.google.com/storage/[Google Cloud Storage].
+
+. Install the {TempoOperator}:
+
+.. Create a subscription.
++
+[source,yaml]
+----
+$ oc apply -f - << EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: tempo-product
+  namespace: openshift-tempo-operator
+spec:
+  channel: stable
+  installPlanApproval: Automatic
+  name: tempo-product
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+EOF
+----
+
+.. Check the Operator status.
++
+[source,terminal]
+----
+$ oc get csv -n openshift-tempo-operator
+----
+
+. Run `oc apply -f` to create a Secret for your object storage bucket.
++
+[TIP]
+====
+[source,terminal]
+----
+$ oc apply -f <secret_file>.yaml
+----
+
+[source,terminal]
+----
+$ oc apply -f - << EOF
+<object_storage_secret>
+EOF
+----
+====
++
+--
+include::snippets/distr-tracing-tempo-required-secret-parameters.adoc[]
+--
++
+--
+include::snippets/distr-tracing-tempo-secret-example.adoc[]
+--
+
+. Run `oc new-project` to create an OpenShift project of your choice for the *TempoStack* instance that you will create in the next step.
++
+[TIP]
+====
+[source,terminal]
+----
+$ oc new-project <openshift_project_of_tempostack_instance>
+----
+
+[source,terminal]
+----
+$ oc apply -f - << EOF
+apiVersion: project.openshift.io/v1
+kind: Project
+metadata:
+  labels:
+    kubernetes.io/metadata.name: <openshift_project_of_tempostack_instance>
+    openshift.io/cluster-monitoring: "true"
+  name: <openshift_project_of_tempostack_instance>
+EOF
+----
+====
+
+. Create a *TempoStack* instance in the OpenShift project that you created for the *TempoStack* instance in the previous step.
++
+NOTE: You can create multiple *TempoStack* instances in separate projects on the same cluster.
```

Comment:
we can move this note out of this procedure

Reply:
Thank you for the suggestion. I already mentioned this possibility at the assembly level, so this is just an additional reminder.